deployment/api-deployment.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name: api
+    app.kubernetes.io/component: api
+    app.kubernetes.io/part-of: monocl
+  name: api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: api
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: api
+    spec:
+      containers:
+        - args:
+            - monocle-api
+            - --port
+            - "9898"
+          env:
+            - name: CONFIG
+              value: /etc/monocle/config.yaml
+            - name: ELASTIC_CONN
+              value: elastic:9200
+          image: quay.io/change-metrics/monocle_api:latest
+          livenessProbe:
+            exec:
+              command:
+                - python -c "import requests,sys; r=requests.post('http://localhost:9898/api/2/health', json={}); print(r.text); sys.exit(1) if r.status_code!=200 else sys.exit(0)"
+            failureThreshold: 6
+            timeoutSeconds: 60
```

Comment:
This is actually not working, the pod is in error because:

```
Warning  Unhealthy  9s                 kubelet            Liveness probe errored: rpc error: code = Unknown desc = failed to exec in container: failed to start exec "5088ef42a6771ed8fb218fd6500d1d7533d1271187e0e1ef4f010ca6333e9f52": OCI runtime exec failed: exec failed: container_linux.go:380: starting container process caused: exec: "python -c \"import requests,sys; r=requests.post('http://localhost:9898/api/2/health', json={}); print(r.text); sys.exit(1) if r.status_code!=200 else sys.exit(0)\"": stat python -c "import requests,sys; r=requests.post('http://localhost:9898/api/2/health', json={}); print(r.text); sys.exit(1) if r.status_code!=200 else sys.exit(0)": no such file or directory: unknown
```

Let's remove it for now.
```suggestion
```